authors/apps/profiles/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from .views import ProfileUpdateAPIView, ProfileGetAPIView
+
+app_name = 'profiles'
+
+urlpatterns = [
+    path('profiles/', ProfileGetAPIView.as_view(), name='get_profiles'),
+    path('profiles/<str:username>/', ProfileUpdateAPIView.as_view(), name='user_profile')
```

Comment:
This line is too long. Try shortening it

Reply:
I have not yet added flake8 standards because the `tox PR` isn't merged into `develop`. When it's merged, I will rebase and correct all the linting errors available.